db/migrate/20231201104328_add_name_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddNameToUsers < ActiveRecord::Migration[7.1]
+  def change
+    add_column :users, :name, :string
```

Comment:
Our models have a validation ensuring that we always set the user's name. There is no matching validation in the database schema. If somebody accessed the database, they could create users without a name. This would corrupt our data. A good practice is to reflect the model validations in our database.